Source/FullScreenPokemon.ts

```diff
@@ -2775,6 +2775,16 @@ module FullScreenPokemon {
                     }
                 },
                 steps * speed);
+
+            // allow the user to move the character after the jump 
+            thing.FSP.TimeHandler.addEvent(
+                function (): void {
+                    if (thing.player) {
+                        (<IPlayer>thing).canKeyWalking = true;
+                        thing.FSP.MapScreener.blockInputs = false;
+                    }
+                },
+                steps * speed);
```

Comment:
Why add a separate event for if the `steps * speed` delay is the same as deleting the shadow?